apps/chat/src/store/ui/ui.epics.ts

```diff
@@ -287,6 +287,17 @@ const resizeEpic: AppEpic = (action$, state$) =>
           }
         }
       }
+
+      if (isMediumScreen()) {
+        if (
+          [showChatbar, showPromptbar].filter(Boolean).length > 1 // more then one panel open for the medium screen)
+        ) {
+          if (showChatbar) {
```

Comment:
it's extra check - you already checked it here
`[showChatbar, showPromptbar].filter(Boolean).length > 1`

Reply:
Changed.
Thank you for the suggestion!